components/third_party_licenses.csv

```diff
@@ -194,3 +194,10 @@ grpc-google-logging-v2,https://raw.githubusercontent.com/googleapis/googleapis/m
 grpc-google-pubsub-v1,https://raw.githubusercontent.com/googleapis/googleapis/master/LICENSE,Apache 2.0
 google-cloud-datastore,https://raw.githubusercontent.com/GoogleCloudPlatform/google-cloud-datastore/master/LICENSE,Apache 2.0
 pymongo,https://raw.githubusercontent.com/mongodb/mongo-python-driver/master/LICENSE,Apache 2.0
+google-auth-oauthlib,https://raw.githubusercontent.com/googleapis/google-auth-library-python-oauthlib/master/LICENSE,Apache 2.0
+google-pasta,https://raw.githubusercontent.com/google/pasta/master/LICENSE,Apache 2.0
+Keras-Preprocessing,https://raw.githubusercontent.com/keras-team/keras-preprocessing/master/LICENSE,MIT
```

Comment:
Curious where keras is used?

Reply:
Unfortunately, the `ml-metadata` package depends on the `tensorflow` package. I filed a bug to unbundle it: https://github.com/google/ml-metadata/issues/25

Reply:
Oh I see.